src/targetPacks/ILsrc/microsoft.aspnetcore.app.ref/3.1.8/Microsoft.AspNetCore.App.Ref.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <OutputPath>$(ArtifactsBinDir)microsoft.netcore.app/3.1.8/</OutputPath>
+    <BaseIntermediateOutputPath>$(ArtifactsObjDir)microsoft.netcore.app/3.1.8/</BaseIntermediateOutputPath>
```

Comment:
Wrong project name. Adopt new project style from text only packages that doesn't require manual fixups?

https://github.com/dotnet/source-build-reference-packages/blob/a72f1ee968104a9c13622e0da4ca42fa1fe9e523/src/textOnlyPackages/src/microsoft.dotnet.common.itemtemplates/3.1.8/Microsoft.DotNet.Common.ItemTemplates.csproj#L1-L12

Reply:
Thanks.  Again, updated all projects under Microsoft.AspNetCore.App.Ref.